network/src/commonTest/kotlin/com/wire/kalium/api/tools/json/api/notification/NotificationEventsResponseJson.kt

```diff
@@ -53,6 +82,12 @@ object NotificationEventsResponseJson {
               ],
               "id": "f484ad9b-8037-11ec-8001-22000a0fe467"
             },
+            {
+              "payload": [
+                ${mlsWelcome.rawJson}
+              ],
+              "id": "7e676173-b715-11ec-8001-22000a252765"
+            },
```

Comment:
Is this enough to test the decoding? I just repeated what was done for the `user.client-add` event.